src/components/layout/NavBar.vue

```diff
@@ -23,6 +23,9 @@
           <router-link v-if="botStore.canRunBacktest" class="nav-link navbar-nav" to="/backtest"
             >Backtest</router-link
           >
+          <router-link class="nav-link navbar-nav" to="/pairlist_config"
```

Comment:
I'm unclear on our assumption for this at the moment
do we want to have this only available in webserver mode - or also in trade mode?

it technically works in both - but in trade mode, it CAN cause ratelimit issues for the "running" exchange class, negatively impacting the actual trading - as the 2 don't know of each other - so we might want to limit it to webserver mode for now ? (thougts?)

Reply:
I wasn't sure either and potential rate limit issues were the only issue I could see with having it available for both as well. Probably safer to limit it to webserver for now yeah.